examples/basic/__main__.py

```diff
@@ -13,11 +16,40 @@
 traceback(app)
 
 
+cheering = os.path.join(os.path.dirname(__file__), 'crowd-cheering.mp3')
+
+
+async def file_iterator(path):
```

Comment:
Don't we want that one as an extension somehow?

Reply:
https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.loop.sendfile ?